### ISSUE TITLE
[ ERROR ] Failed to execute stage 'Misc configuration': [Errno 2] No such file or directory: '/usr/share/ovirt-engine/services/ovirt-engine/keycloakxm_h2n4m/keycloak-add-user.json'

### ISSUE THREAD

**Issue by iliv** [oVirt/ovirt-engine-keycloak#60]
I ran into this problem trying to install oVirt Engine 4.5.4 in Standalone Mode on vanilla CentOS 8 Stream.

```
[ ERROR ] Failed to execute stage 'Misc configuration': [Errno 2] No such file or directory: '/usr/share/ovirt-engine/services/ovirt-engine/keycloakxm_h2n4m/keycloak-add-user.json'
```

I tried just about everything I could think of including removing ALL oVirt related packages (even repository packages) and re-installing them again. Rebooting the host between removing of the packages and reinstalling them, etc. 

Nothing helped.

The only way I could make engine-setup happy (or whatever it is under the hood) was to provide different passwords for Engine admin and Keycloak admin users. 

Basically, this:

```
Use Engine admin password as initial keycloak admin password (Yes, No) [Yes]:No
          Keycloak [admin] password:
          Please confirm password:
```

If I answered Yes to "Use Engine admin password as initial keycloak admin password (Yes, No)", I would consistently run into the problem cited above.

**Comment by artbird309** [comment]
I had this same issue, I removed the space from my password and I was able to use the same password for Keycloak and the Engine.

**Comment by iliv** [comment]
Well, there was no space symbol in my password.

**Comment by Twilight-Computer** [comment]
I can confirm that using a password with a space causes this error to appear. Removed oVirt and re-installed. Ran engine-setup using a password with no spaces and installation completed without the error.
